package.json

```diff
@@ -113,6 +113,7 @@
     "live-server": "^1.2.1",
     "live-server-https": "^0.0.2",
     "nightwatch": "1.2.4",
+    "node-fetch": "2.6.0",
```

Comment:
remove

Reply:
I realized that you only need to regenerate the fragment for union types if you add/remove a type used in a union. You don't need to regenerate it if you only add/remove fields to those types.